packages/nextjs/app/builders/0x6d3321b4637D8451740Cf400848602f838F7A594/page.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { FaGithub } from "react-icons/fa";
+import { SiGmail } from "react-icons/si";
+import { Address } from "~~/components/scaffold-eth";
+import { BlockieAvatar } from "~~/components/scaffold-eth";
+
+const PersonalPage = () => {
+  return (
+    <div className=" flex justify-center items-center">
+      <div>
+        <div className=" pt-24 lg:pt-[100px]">
+          <div className="flex flex-col-reverse lg:flex-row items-center justify-center gap-x-[50px] md:mx-[550px]">
+            <div className=" w-full">
+              <h1 className=" text-[40px] font-bold md:my-[1rem]">Hello 👋🏼</h1>
+              <p className=" text-[14px] md:text-lg p-1 font-semibold w-full text-justify mb-[6px]">
+                My name is Abdulkadir. I want to become a doctor in the future. I really like computer games. I try to
+                balance my studies with my hobbies.
+              </p>
+              <div className=" flex items-center gap-x-2 -my-3">
+                <p>Address :</p>{" "}
```

Comment:
<img width="398" alt="Screenshot 2024-09-26 at 18 24 35" src="https://github.com/user-attachments/assets/43897e98-9444-45e6-b6fc-1b3361763eac">

It looks like the : is appearing on the next line (on smaller screen sizes). It would be great to have it in on line with Address.

And doublecheck if you need `{" "}`